src/MachO/BinaryParser.tcc

```diff
@@ -297,7 +297,16 @@ void BinaryParser::parse_load_commands(void) {
           load_command = new LoadCommand{command};
           break;
         }
-
+      case LOAD_COMMAND_TYPES::LC_ENCRYPTION_INFO_64:
```

Comment:
I started the project with the requirement to parse **everything** in formats including unused fields.
In other formats ELF/PE we also have values that are not used. Depending on the situation It can be useful to know this value. (Malware etc)